### ISSUE TITLE
Website - Host on GitHub

### ISSUE THREAD

**Issue by bryankennedy** [cns-iu/xmacroscope#248]
In the interest of expediency, I started out hosting our site on the SMM setup. There's an interest in hosting the site on GitHub.

To do that we need to figure out the SSL setup. Here's a explanation of how to do that. Not sure if that works on non GH-pages stuff, but I'm saving it here as a bookmark.
https://blog.github.com/2018-05-01-github-pages-custom-domains-https/

**Comment by bherr2** [comment]
Yeah, should totally be doable to host on GitHub. I've looked into it before but never had an opportunity to actually try it. Let me know if you need some help/input. When it starts working, we can pretty easily update Travis to build it and the demo.

**Comment by bryankennedy** [comment]
I'm moving this back for now. We're going to need to integrate the website platform with some of our internal deploy process to get the data dashboard working. While this should be technically possible on Github, we need to use our existing deploy mechanisms to get it setup first. At a later date we can evaluate whether it's worth moving to Github.
